src/core/thread/thread_uri_paths.hpp

```diff
@@ -0,0 +1,346 @@
+
+/**
+ * @def OT_URI_PATH_LEADER_PETITION
+ *
+ * The URI Path for Leader Petition
+ *
+ */
+#define OT_URI_PATH_LEADER_PETITION "c/lp"
+
+/**
+ * @def OT_URI_PATH_LEADER_KEEP_ALIVE
+ *
+ * The URI Path for Leader Keep Alive
+ *
+ */
+#define OT_URI_PATH_LEADER_KEEP_ALIVE "c/la"
+
+/**
+ * @def OT_URI_PATH_PANID_CONFLICT
+ *
+ * The URI Path for PAN ID Conflict
+ *
+ */
+#define OT_URI_PATH_PANID_CONFLICT "c/pc"
+
+/**
+ * @def OT_URI_PATH_PANID_QUERY
+ *
+ * The URI Path for PAN ID Query
+ *
+ */
+#define OT_URI_PATH_PANID_QUERY "c/pq"
+
+/**
+ * @def OT_URI_PATH_COMMISSIONER_GET
+ *
+ * The URI Path for MGMT_COMMISSIONER_GET
+ *
+ */
+#define OT_URI_PATH_COMMISSIONER_GET "c/cg"
+
+/**
+ * @def OT_URI_PATH_COMMISSIONER_KEEP_ALIVE
+ *
+ * The URI Path for Commissioner Keep Alive.
+ *
+ */
+#define OT_URI_PATH_COMMISSIONER_KEEP_ALIVE "c/ca"
+
+/**
+ * @def OT_URI_PATH_COMMISSIONER_PETITION
+ *
+ * The URI Path for Commissioner Petition.
+ *
+ */
+#define OT_URI_PATH_COMMISSIONER_PETITION "c/cp"
+
+/**
+ * @def OT_URI_PATH_COMMISSIONER_SET
+ *
+ * The URI Path for MGMT_COMMISSIONER_SET
+ *
+ */
+#define OT_URI_PATH_COMMISSIONER_SET "c/cs"
+
+/**
+ * @def OT_URI_PATH_DIAGNOSTIC_GET_REQUEST
+ *
+ * The URI Path for Network Diagnostic Get Request.
+ *
+ */
+#define OT_URI_PATH_DIAGNOSTIC_GET_REQUEST "d/dg"
+
+/**
+ * @def OT_URI_PATH_DIAGNOSTIC_GET_QUERY
+ *
+ * The URI Path for Network Diagnostic Get Query.
+ *
+ */
+#define OT_URI_PATH_DIAGNOSTIC_GET_QUERY "d/dq"
+
+/**
+ * @def OT_URI_PATH_DIAGNOSTIC_GET_ANSWER
+ *
+ * The URI Path for Network Diagnostic Get Answer.
+ *
+ */
+#define OT_URI_PATH_DIAGNOSTIC_GET_ANSWER "d/da"
+
+/**
+ * @def OT_URI_PATH_DIAG_RST
+ *
+ * The URI Path for Network Diagnostic Reset.
+ *
+ */
+#define OT_URI_PATH_DIAGNOSTIC_RESET "d/dr"
+
+/**
+ * @def OT_URI_PATH_MLR
+ *
+ * The URI Path for Multicast Listener Registration.
+ *
+ */
+#define OT_URI_PATH_MLR "n/mr"
+
+/**
+ * @def OT_URI_PATH_DUA_REGISTRATION_REQUEST
+ *
+ * The URI Path for Domain Unicast Address Registration Request (DUA.req).
+ *
+ */
+#define OT_URI_PATH_DUA_REGISTRATION_REQUEST "n/dr"
+
+/**
+ * @def OT_URI_PATH_DUA_REGISTRATION_NOTIFY
+ *
+ * The URI Path for Domain Unicast Address Registration Notification (DUA.ntf).
+ *
+ */
+#define OT_URI_PATH_DUA_REGISTRATION_NOTIFY "n/dn"
+
+/**
+ * @def OT_URI_PATH_BACKBONE_QUERY
+ *
+ * The URI Path for Backbone Query (BB.qry).
+ *
+ */
+#define OT_URI_PATH_BACKBONE_QUERY "b/bq"
+
+/**
+ * @def OT_URI_PATH_BACKBONE_ANSWER
+ *
+ * The URI Path for Backbone Answer with destination to a link-local unicast address (BB.ans),
+ * or Proactive Backbone Notification with destination to a link-local multicast address (PRO_BB.ntf).
+ *
+ */
+#define OT_URI_PATH_BACKBONE_ANSWER "b/ba"
+
+/**
+ * @def OT_URI_PATH_BACKBONE_MLR
+ *
+ * The URI Path for Backbone Multicast Listener Report.
+ *
+ */
+#define OT_URI_PATH_BACKBONE_MLR "b/bmr"
+
+} // namespace ot
+
+#endif // THREAD_URIS_HPP_
```

Comment:
https://github.com/openthread/openthread/pull/5482 introduced src/core/thread/uri_paths.hpp for Thread URIs.

Reply:
OMG. I added the whole file back. 👎

Reply:
fixed